Proteomics/Protein/Protein.cs

```diff
@@ -820,6 +820,46 @@ private void AddIntactProteoformToProteolysisProducts(InitiatorMethionineBehavio
             }
         }
 
+        public void CleaveOnceBetweenProteolysisProducts()
```

Comment:
I was starting to get concerned about performance of the proteolysis product generation. So I measured performance of CleaveOnceBetweenProteolysisProducts as used in the unit test in TestDigestionMotifs: method plus subsequent .Select().ToList() is .176 ms, so performance seems good to me.